2019/d01p1FuelCounterUpper.d

```diff
@@ -0,0 +1,7 @@
+unittest {
+  import std.algorithm;
```

Comment:
Are we supposed to ignore this line?

Reply:
`map` and `sum` are algorithms, where else would one get them from but the standard algorithms library?